mutagen/ac3.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019 Philipp Wolfer
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+
+"""Pure AC3 file information.
+"""
+
+__all__ = ["AC3", "Open"]
+
+from mutagen._compat import endswith
+from mutagen._file import FileType
+from mutagen._util import (
+    BitReader,
+    BitReaderError,
+    MutagenError,
+    convert_error,
+    loadfile,
+)
+
+
+AC3_HEADER_SIZE = 7
+
+AC3_CHMODE_DUALMONO = 0
```

Comment:
mutagen includes a simple enum type btw, example: https://github.com/quodlibet/mutagen/blob/4a5d7d17f1a611280cc52d229aa70b77ca3c55dd/mutagen/id3/_specs.py#L21

Reply:
Good thing to know. Used this fo ChannelMode and EAC3FrameType, the code reads better now.